easybuild/easyconfigs/z/zlib/zlib-1.2.7-goolf-1.7.20.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'zlib'
+version = '1.2.7'
```

Comment:
@npklein: do we need to keep this one around?

If you remove it, also adjust the PR title please.